stdlib/public/runtime/HeapObject.cpp

```diff
@@ -229,13 +229,15 @@ static Lazy<MetadataCache<BoxCacheEntry>> Boxes;
 
 SWIFT_RUNTIME_EXPORT
 BoxPair::Return
-swift::swift_allocBox(const Metadata *type) {
+swift::swift_allocBox(const Metadata *type)
+    SWIFT_CC(swift) {
   return SWIFT_RT_ENTRY_REF(swift_allocBox)(type);
 }
 
 SWIFT_RT_ENTRY_IMPL_VISIBILITY
 extern "C"
-BoxPair::Return SWIFT_RT_ENTRY_IMPL(swift_allocBox)(const Metadata *type) {
+BoxPair::Return SWIFT_RT_ENTRY_IMPL(swift_allocBox)(const Metadata *type)
+    SWIFT_CC(swift) {
```

Comment:
I thought the dominant style is to put `SWIFT_CC(swift)` on a separate line before the declaration -- doesn't that work in this case?